experiments/experiments-impl/src/main/java/com/duckduckgo/experiments/impl/VariantManagerImpl.kt

```diff
@@ -106,12 +106,20 @@ class VariantManagerImpl @Inject constructor(
         if (entity.key == "sc" || entity.key == "se") {
             return { isSerpRegionToggleCountry() }
         }
-        if (entity.localeFilter.isEmpty()) {
-            return { noFilter() }
+
+        var matchesLocaleFilter = true
```

Comment:
This can all be simplified to a couple of lines

```
val matchesLocale = entity.localeFilter.contains(Locale.getDefault().toString())
val matchesAndroidVersion = entity.androidVersionFilter.contains(appBuildConfig.sdkInt.toString())
```

Reply:
It needs to be true by default though

Reply:
gotcha